pywbem/cim_operations.py

```diff
@@ -486,6 +487,32 @@ class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
     the :attr:`last_raw_request` and :attr:`last_raw_reply` attributes
     of the connection object.
 
+    WBEMConnections can record the results of methods that interact with
+    a WBEM Server using either the :class:`~pywbem.LogOperationRecorder` (uses
+    python logging
+    to create logs for methods and http data) or the
+    :class:`~pywbem.TestClientRecorder`( creates a yaml file of methods and
+    http for tests) which are subclasses of the :class:`~pywbem.Recorder`
+    class.
+
+    Before pywbem 0.11.0, there was only a single operation recorder and
+    activating/deactivating the recorder was a simple matter of setting the
+    value of an instance of the :class:`~pywbem.TestClientRecorder` into the
+    WBEMConnection.operation_recorder attribute.
+    Setting the value of TestClientRecorder into this
+    attribute activated and enabled the recorder.  Setting it to None
+    deactivated the recorder.
+
+    Effective version 0.11.0, pywbem added a second recorder and the requirement
+    that both recorders operate simultaneously. Thus, if both are enabled and
+    active, operation data can be both recorded to a yaml file and to the log
+    simultaneously.
+
+    The method to activate has therefore change. To activate a rcorder you must
+    use the WBEM connection method:
+
+        `:class:`~pywbem.add_operationsrecorder`
```

Comment:
incorrect method name

Reply:
Fixed but per previous comment, this or the version in clients.rst will disappear

Reply:
TODO